yarn.lock

```diff
@@ -2350,6 +2350,8 @@ pkg-dir@^4.1.0, pkg-dir@^4.2.0:
   dependencies:
     find-up "^4.0.0"
 
+<<<<<<< HEAD
+=======
```

Comment:
Looks like this was accidentally committed

Reply:
I'm surprised a precommit check didn't catch this

Reply:
Yeah, we have a pre-commit check that runs during `git merge` so the only time it would have run was during the merge commit above. @pcvera is it possible you don't have `pre-commit` hooks installed in your local git repo?

In any case, I'll put up a PR to enable merge conflict lint checking always so we can catch these in Github :+1: Thanks for pointing it out, @erwa!

Reply:
See #247 for the fix